keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -514,13 +518,6 @@ def compute_metrics(self, x, y, y_pred, sample_weight):
                 metrics[metric.name] = result
         return metrics
 
-    @classmethod
-    def from_config(cls, config):
-        config["backbone"] = keras.utils.deserialize_keras_object(
-            config["backbone"]
-        )
-        return super().from_config(config)
-
```

Comment:
In `get_config` should the other objects after `backbone` be serialized by default as well? We can file an issue to clean this up later if so; custom objects don't seem to be tested.

Reply:
the short answer is I don't know, but this was breaking the tests and I dont think this is the proper approach. 

I think we should explicitly address from_config() and get_config() in a separate PR.

Reply:
the short answer is I don't know, but this was breaking the tests and I dont think this is the proper approach. 

I think we should explicitly address from_config() and get_config() in a separate PR.